gtsfm/bundle/bundle_adjustment.py

```diff
@@ -195,6 +260,18 @@ def __optimize_factor_graph(self, graph: NonlinearFactorGraph, initial_values: V
         result_values = lm.optimize()
         return result_values
 
+    def __cameras_to_model(
+        self,
+        initial_data: GtsfmData,
+        absolute_pose_priors: List[Optional[PosePrior]],
+        relative_pose_priors: Dict[Tuple[int, int], Optional[PosePrior]],
+    ) -> List[int]:
+        """Get the cameras which are to be modelled in the factor graph. We are using ability to add initial values as
```

Comment:
"modeled". But I don;t understand this function

Reply:
This is to figure out what cameras we will model in the factor graph.

For example: lets say we only receive 90 of the 100 input cameras after averaging and data association. If we do have prior information available on the remaining 10 cameras (lets assume the priors connect them to the clique of 90 cameras). Should we add them to the factor graph? 

Right now I am keeping the logic the same, i.e. add only those cameras which are part of a track. But created a function out of it for experimentation.